src/examples/typography/span/Hue.tsx

```diff
@@ -19,9 +19,9 @@ const Example = () => (
       <MD tag="span">
         Water spinach arugula pea tatsoi aubergine spring onion bush tomato kale radicchio turnip
         chicory salsify pea sprouts fava bean.{' '}
-        <Span hue="fuschia">Dandelion zucchini burdock yarrow chickpea dandelion</Span> sorrel
-        courgette turnip greens tigernut soybean radish artichoke wattle seed endive groundnut
-        broccoli arugula.
+        <Span hue="foreground.primary">Dandelion zucchini burdock yarrow chickpea dandelion</Span>{' '}
+        sorrel courgette turnip greens tigernut soybean radish artichoke wattle seed endive
+        groundnut broccoli arugula.
```

Comment:
Should we show all 3 possibilities for setting the text color: variables, hue, and HEX?

Reply:
I'm open to it. I figured we didn't want to just because we encourage avoiding hard coded hexes.

Reply:
For the purposes of this PR, let's leave the essential content as-is. We can look at example enhancements later. But we probably don't want to encourage random hex usage, in general.